frontend/src/components/account/ChangePassword/ChangePasswordForm.module.scss

```diff
@@ -28,3 +28,15 @@
   gap: 32px;
   margin-top: 32px;
 }
+
+.passwordReq {
+  color: $gray-dark-color;
```

Comment:
Please, add some space between this hint and the input

Reply:
There is no space according to the design (you can check design at registration modal)